docs/help/getting-started.md

```diff
@@ -1,27 +1,56 @@
-Step 1: Create your space/documentation
-Step 2: Upload your file
-    - upload file directly
-    - use the CLI
-    - try an example
-Step 3: Enjoy
+
+### Step 1: Create your space/documentation
+
+After creating your account, you'll be asked to name your first API documentation.
+By default, your documentation URL will use the same name: you can customize it in the slug field. You can also use [custom domains](help/custom-domains.md), which we'll explain later.
+
+![](/files/doc-creation.png)
+
+### Step 2: Upload your file
+
+If you already have a specification file, you can immediately upload it.
+You can also use our examples to see Bump.sh in action. Finally, our [CLI](help/bump-cli) can also be directly used to jump in.
+
+![](/files/upload-spec-file.png)
+
+### Step 3: Enjoy
+
+Generating your first documentation will take a few seconds. But let's not stop here: many features are awaiting to customize and make the most of your API documentation. Let's have a look in the next section.
+
 
 ## Manage Access
-- Make it public or private
-- Create an organization
-- Invite teammates
+
+Congratulations on releasing your first API documentation on Bump.sh!
+
+One of the first things you may have in mind is how to share your documentation with the world, your team or keep it for yourself.
+Bump.sh offers several options to manage access to your documentation.
+During the onboarding, you may have noticed the option of making your doc public or private.
+
+Public documentation are available to anyone with the URL or if the search engine indexation has been activated.
+Privates are only accessible to you (or your team/organization if you have one).
+
+We share more details on the [Access Management page](help/access-management.md).
 
 ## Customize it
-- Branding options
-- Markdown support
-- Custom domains
-
-## Going further
-- Hubs
-- Branching
-- Webhooks
-... much more.
\ No newline at end of file
+
+With your documentation ready, you could want to make it more customized. Bump.sh offers several options to do so.
+
+You can setup [Custom Domains](help/custom-domains.md), to make your doc available behind your own URL.
+Several branding options are available: from your logo to [Meta Images](help/meta-images.md) (that helps sharing your doc in a fancy way).
+We also support [Documentation Topics](help/doc-topics.md), that helps you adding rich (con)text to your API doc, sharing the informations your community needs.
+
+![](/files/meta-image-2.png)
+
+## And way much more
+
+We do not stop here. Feel free to browse this documentation or directly reach us to find out how to make the most of Bump.sh.
+There's a lot more to cover: from Hubs that groups several API documentation to Branching, hosting several version of a same doc, you'll find many simple options that make a huge change.
+
+The team is always here to listen if you have any feature requests or suggestions. We are here to help.
+
+### We hope you'll enjoy using Bump.sh!
```

Comment:
Feels strange to use a “Title 3” when there's no texte behind. What about using only bold?

```suggestion
**We hope you'll enjoy using Bump.sh!**
```